opm/simulators/timestepping/TimeStepControl.hpp

```diff
@@ -65,6 +65,8 @@ namespace Opm
                                    const int iterations,
                                    const RelativeChangeInterface& /* relativeChange */,
                                    const AdaptiveSimulatorTimer& /* substepTimer */ ) const override;
+        
+        bool timeStepAccepted(const double error) const override { return true; }
```

Comment:
Please tag the `error` parameter as `[[maybe_unused]]`&ndash;or remove the name altogether&ndash;so as to ensure we do not get [additional warnings](https://ci.opm-project.org/job/opm-simulators-PR-builder/8108/gcc/) when this PR gets merged.